go/chat/utils/utils.go

```diff
@@ -962,19 +962,33 @@ func PresentMessageUnboxed(ctx context.Context, g *globals.Context, rawMsg chat1
 	}
 	switch state {
 	case chat1.MessageUnboxedState_VALID:
+		var valid chat1.MessageUnboxedValid
 		if !rawMsg.IsValidFull() {
-			return miscErr(fmt.Errorf("unexpected deleted %v message",
-				strings.ToLower(rawMsg.GetMessageType().String())))
+			showErr := true
+			if rawMsg.IsValid() {
```

Comment:
I've seen this pattern a lot, can we pull it out into its own function?

Reply:
The logic is different in the other place we use this so I don't think it would help much: 

```
supersedes.go:191:                                      if !mvalid.IsExploding() || mvalid.HideExplosion(time.Now()) {
utils/utils.go:477:                     if mvalid.IsExploding() && mvalid.HideExplosion(now) {
utils/utils.go:972:                             if valid.IsExploding() && valid.IsEphemeralExpired(time.Now()) {
```

Reply:
At the very least you don't need to check `IsValid()` again, this is in a block where it has already been determined this messages is valid.

Reply:
sounds good!